package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
+    "test": "echo \"No tests yet\" && exit 0",
```

Comment:
Please remove the test script, we don't have tests in our project therefore we don't need it